.coveragerc

```diff
@@ -106,6 +106,9 @@ omit =
     homeassistant/components/hive.py
     homeassistant/components/*/hive.py
 
+    homeassistant/components/homekit_controller.py
```

Comment:
Please update the path to point at the `__init__.py` file under `homekit_controller` component directory.